BicycleParkingStationForecast/examples/example-normalized.jsonld

```diff
@@ -29,7 +29,7 @@
         ]
     },
     "@context": [
-        "https://raw.githubusercontent.com/smart-data-models/dataModel.OSLO/master/context.jsonld",
+        "https://raw.githubusercontent.com/smart-data-models/dataModel.OSLO/master/notes_context.jsonld",
```

Comment:
initially wrong because notes_context.jsonld is only for customization and the @context includes the preventions included in the notes_context.jsonld